openapi/openapi.yaml

```diff
@@ -1792,6 +1794,10 @@ components:
                 enum:
                   - BUILDTIME
                   - RUNTIME
+              package_extract_document_id:
+                type: string
+              datetime:
+                type: string
```

Comment:
btw I'm not sure if this gets properly serialized to a string

Reply:
should we adjust this on thoth-storages side?

Reply:
We can add `format_datetime` from thoth common

Reply:
https://github.com/thoth-station/storages/pull/2512/commits/eb3a980c78e9a431d0a12232cc1b334b7915efc7, wdyt @fridex ?

Reply:
> We can add `format_datetime` from thoth common

+1

> should we adjust this on thoth-storages side?

regarding the serialization - it can be done on thoth-storages side to be consistent with the rest of the code:

https://github.com/thoth-station/storages/blob/bf153d60fede2e187d562172c50543016462d5c3/thoth/storages/graph/postgres.py#L563

Reply:
Done https://github.com/thoth-station/storages/pull/2512/commits/4090f9df7db87fb5df2e334152c60d8803c87cf8, Thanks @fridex !! 🍻